SDK/AppCenterData/Microsoft.AppCenter.Data.iOS/ConversionExtensions.cs

```diff
@@ -15,19 +15,30 @@ public static class ConversionExtensions
 
         public static DocumentWrapper<T> ToDocumentWrapper<T>(this MSDocumentWrapper documentWrapper)
         {
-            var jsonValue = documentWrapper.JsonValue;
-            var deserializedValue = jsonValue != null ? JsonConvert.DeserializeObject<T>(jsonValue) : default(T);
-            return new DocumentWrapper<T>
+            DocumentWrapper<T> parsedDocument = new DocumentWrapper<T>
             {
-                DeserializedValue = deserializedValue,
-                JsonValue = jsonValue,
                 Partition = documentWrapper.Partition,
                 ETag = documentWrapper.ETag,
                 Id = documentWrapper.DocumentId,
-                LastUpdatedDate = documentWrapper.LastUpdatedDate != null ? (DateTime)documentWrapper?.LastUpdatedDate : UnixEpoch,
                 IsFromDeviceCache = documentWrapper.FromDeviceCache
             };
-        }
+
+            try
+            {
+                var jsonValue = documentWrapper.JsonValue;
```

Comment:
why is jsonValue inside the try/catch block? we should expose it even if deserializing fails.